src/pages/home.js

```diff
@@ -25,6 +26,26 @@ export default function Home() {
             </Link>
           </div>
         </div>
+        <Particles
```

Comment:
The particles look fine. Could you reduce their density and make them the background of the hero?

Reply:
Yes I can reduce the density, and I will figure out how to make it the background as well.